docs/DeveloperGuide.md

```diff
@@ -972,6 +1037,40 @@ Priorities: High - must have; Medium - nice to have;  Low - unlikely to have.
     
 <br/>
 
+**Use case: UC12 – View reminder**
+
+**MSS**
+
+1. User requests for reminder.
+2. TAB displays a list of upcoming lessons.
+
+Use case ends.
+
+**Extension**
+
+* 1a. There are no upcoming lessons.
```

Comment:
```suggestion
* 1a. There are no reminders.
```

Reply:
i think there are no upcoming lessons is more accurate. "reminder" is just the window